pkg/certificate/triple/cert.go

```diff
@@ -154,8 +161,8 @@ func GenerateSelfSignedCertKey(host string, alternateIPs []net.IP, alternateDNS
 // <host>_<ip>-<ip>_<alternateDNS>-<alternateDNS>.key
 // Certs/keys not existing in that directory are created.
 func GenerateSelfSignedCertKeyWithFixtures(host string, alternateIPs []net.IP, alternateDNS []string, fixtureDirectory string) ([]byte, []byte, error) {
-	validFrom := time.Now().Add(-time.Hour) // valid an hour earlier to avoid flakes due to clock skew
-	maxAge := time.Hour * 24 * 365          // one year self-signed certs
+	validFrom := Now().Add(-time.Hour) // valid an hour earlier to avoid flakes due to clock skew
```

Comment:
general question. by making the cert valid an hour earlier then "now" - couldn't it make it so that 2 certs are valid in the same time?

Reply:
I copy pasted it from kubevirt but we don't need this functions, removing them, thanks !